README.md

```diff
@@ -66,17 +70,31 @@ In order to get started quickly, you can deploy the docker image to a DigitalOce
 
 *Disclaimer*: The DigitalOcean server is publicly accessible on the Internet. Do not put sensitive information on the network that you would not want someone else to know. Anyone with access to the network will be able to use the root account above.
 
-### Soroban RPC Server
+### Soroban Dev
+
+For local development on smart contracts on stellar(aka soroban), it is recommended to run the soroban stack locally via the `stellar/quickstart:soroban-dev` image:
+
+```
+$ docker run --rm -it \
+    -p "8000:8000" \
+    --name stellar \
+    stellar/quickstart:soroban-dev \
+    --futurenet \
```

Comment:
I think at this stage we still recommend `--standalone`?

Reply:
ok, I'll change, using `standalone` is part of the expected [dapp workflow](https://github.com/stellar/soroban-example-dapp#user-workflows) correct? Do we need to adjust the `standalone` core cfg in any relation to soroban support or is ok as-is?

Reply:
Standalone core cfg should be fine as is afaik.